### DISCUSSION TITLE
Compose Image size before making request

### DISCUSSION THREAD

**Discussion by CurtJRees** [coil-kt/coil#849] [Q&A]
Are there are plans to provide a size to the request builder, similar to how Accompanist was handling it:
 `requestBuilder: (ImageRequest.Builder.(size: IntSize) -> ImageRequest.Builder)? = { this },`

Or possibly a Compose SizeResolver?
At the moment I'm having trouble getting the width of an Image composable so I can build a url with the size included

**Comment by colinrtwhite** [comment]
No plans for this - I'd recommend using an `Interceptor` for this similar to the [Jetpack Compose samples](https://github.com/android/compose-samples/blob/main/Crane/app/src/main/java/androidx/compose/samples/crane/util/UnsplashSizingInterceptor.kt#L33).
